projects/netgrif-components/src/lib/admin/ldap-group-role-assignment/ldap-group-role-assignment.component.html

```diff
@@ -22,10 +22,10 @@
                 </div>
 
                 <span style="flex: 1 1 auto;"></span>
-                <button mat-icon-button matTooltip="Select All" (click)="selectAllLdapGroup(true)">
+                <button mat-icon-button matTooltip="{{ 'side-menu.tooltip.SelectAll' | translate }}" (click)="selectAllLdapGroup(true)">
```

Comment:
why is the tooltip text saved under the `side-menu` key if it is not used in the side menu?

Reply:
all those components are stored there in side-menu...    IDK

Reply:
my bad I mixed up the names of the panel on the left and ride side of the screen....